### ISSUE TITLE
Please consider adding the new optimisation for 1060 (+15%) on ETH !

### ISSUE THREAD

**Issue by pducharme** [DillonN/NiceHashMiner#20]
Have a look there, it is real and no dev Fees!  

https://www.reddit.com/r/EtherMining/comments/6jjhob/ethereum_code_optimized_for_some_nvidian_cards/

**Comment by pducharme** [comment]
It also benefits the 1070 (+3-5%)

**Comment by DillonN** [comment]
Looks like this will implemented into the ethminer repo soon after some tweaks are made (going by the pull request activity). Once it is in there I will look into updating ethminer for NHM. Thanks for bringing it to my attention!

**Comment by pducharme** [comment]
Cool :) Everything I can do to help get back some $/day to NiceHash... it's going down since a week :(   I stopped using your mod when they were a banning spree, but after couple of low low day using official, I went back to your version :)

**Comment by DillonN** [comment]
Yeah mining in general has gone down a lot recently due to the craze. My version should be completely safe (in regards to banning) as long as you use at least 1.7.6.1 and always have a valid BTC address entered. Just make sure not to go to NH for any support since it's unofficial!

**Comment by pducharme** [comment]
Yes, i use the last version and made sure my BTC wallet address was there BEFORE I did the Benchmark.   I'll keep an eye on 1.7.6.2 with that new enhancement, since I currently have 1 x 1060 and 1 x 1070 (with 3 x 1070 coming in this week), that will be great.

**Comment by CreamyG31337** [comment]
Nicehash devs made their own fork of ethminer, you can read about it and get links to the repos here: https://www.nicehash.com/?p=news&id=77
Don't expect this optimization to show up any time soon because:
a) nicehash devs want to add this algo to excavator and work on nicehash 2.x instead. it's probably not high priority for them.
b) you need to use their version if you want to use the gui -- replacing the miner with generic one won't work. --api-port command line option isn't recognozed / backported and i don't know what else is broken.
c) 15% speed improvement would only bring the profit in line with dual mining, so there's no point anyways

**Comment by CreamyG31337** [comment]
they just released a new excavator 1.2.9a with nvidia cuda daggerhashimoto support, and are claiming speeds about 10% faster than what I get with the current ethminer. This might be easier to get working, with similar benefits.

**Comment by CreamyG31337** [comment]
Justed tested this new version vs dual mining --
For me, it gets 23.43 MH/s on my 3GB 1060 @ 65% power limit & +650 MEM clock
If I let Nicehash do it's thing, it wants to do dual mining DaggerSia right now, which Nicehash GUI is estimating at about $2.90 a day if I tweak the intensity as well as I can.
So, I guess this new excavator might be a bit more profitable that dual mining. Not much, though.

**Comment by DillonN** [comment]
Unfortunately Excavator explicitly forbids distributing it, so I cannot update to the newest builds at the moment. There won't be any updates over the next few days but stay tuned for next week!

**Comment by DragosD1** [comment]
Hi Dillon, how do you set up that R290 to pull 6 usd/day? What version and what settings? My R 390 barely pull 3usd/2r h. Thank you
